### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN017.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#63] [General]
# https://blog.3place.cn/3place/lmyn/LMYN017.html

https://blog.3place.cn/3place/lmyn/LMYN017.html

**Comment by picklecai** [comment]
毗湿奴诚实又英勇，
希望为一切生物造福。
请你们给他创造伙伴，
随意变形，勇敢威武。

根据大梵天的要求，就是创造了几百万只猴子？

**Comment by picklecai** [comment]
他们不可抗御，精明强干，
他们具备神仙般的沉着英武，
他们都娴熟所有的兵器，

不可抗御，娴熟兵器，这就是对工具的要求。

**Reply by picklecai** [reply]
罗摩自己，何尝不是这样的工具？

**Comment by picklecai** [comment]
他们样子像山又像象，
非常漂亮，力大无穷，

审美观，是雄性的、壮美。

**Reply by picklecai** [reply]
雄壮的审美，是什么时候渐居下风的？

**Reply by picklecai** [reply]
可以理解为什么菩萨金刚都形象可怕，因为那是雄壮审美视角下的美化。

**Comment by picklecai** [comment]
悉陀、尾儞也陀罗和蛇王，
还有天上歌手都创造了些
猴样的儿子，英勇异常。

这是个啥组合？怎么还有歌手？

**Comment by picklecai** [comment]
他们如果放声一吼，
连天空的鸟也要下降。 

xxx吼三吼，地球也要抖三抖。

**Comment by picklecai** [comment]
许多猴子是牛尾巴猴〔187〕所生，
他们的勇敢大家都公认，
有许多猴子是哩刹所生，
许多猴子以紧那哩〔188〕为母亲。 （1.16.12）

紧那哩，梵文golāṇgūla，意思也是“牛尾”。

看上去牛尾巴猴是猴子中的婆罗门。

**Comment by picklecai** [comment]
尊者自存物大梵天

自存，小农经济的朋友

**Comment by picklecai** [comment]
中国神话中的猴子（西游记、封神演义除外）：

无支祁：中国神话中的水怪，形状像猿猴，塌鼻子，凸额头，白头青身，火眼金睛，力气超过九头大象。
朱厌：出自《山海经》，古代神话中的“战猴”，代表着兵戈和杀伐，它身上的杀气极盛，可以跟上古凶兽白虎相提并论。
觜火猴：二十八星宿，天生的神力，每日接受日月精华的浸润，体内的火精汹涌磅礴，一张嘴就可喷出五色火焰。
参水猿：二十八星宿，感受天地之变化，吸收大海潮汐之能，淬炼江河湖泊水之精，体外总是有一道白茫茫的水精之雾。
